### PR TITLE
MAT-6175 export qdm measure definitions

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-6175](https://jira.cms.gov/browse/MAT-6175)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/resources/templates/humanreadable/measure_information_table.ftl [hunk 1]
```diff
@@ -192,7 +192,13 @@
...
             <td style="width:80%" colspan="3">
                 <div style="width:660px;">
-                    <pre>${model.measureInformation.definition!""}</pre>
+                    <#if model.measureInformation.definitions??>
+            			<#list model.measureInformation.definitions as definition>
+                    			${definition.term!""} - ${definition.definition!""} </br>
+            			</#list>
+        			<#else>
+                			${""}
+        			</#if>
```
Comment (adongare) [inline src/main/resources/templates/humanreadable/measure_information_table.ftl]: not sure if this is required

**Review by adongare — DISMISSED** [review]

File: src/main/resources/templates/humanreadable/measure_information_table.ftl (hunk 1 above)
Comment (sb-cecilialiu) [reply src/main/resources/templates/humanreadable/measure_information_table.ftl]: @adongare Thanks Ashok! Do you think I should remove the else?

Comment (adongare) [reply src/main/resources/templates/humanreadable/measure_information_table.ftl]: if the list is empty anyway the div will be empty. so maybe ~not~ yes.

Comment (sb-cecilialiu) [reply src/main/resources/templates/humanreadable/measure_information_table.ftl]: @adongare ok thanks Ashok, so can I resolve the conversation and merge the code?

Comment (adongare) [reply src/main/resources/templates/humanreadable/measure_information_table.ftl]: Sorry, I meant yes. we can remove `else` part.

Comment (sb-cecilialiu) [reply src/main/resources/templates/humanreadable/measure_information_table.ftl]: @adongare Ok sure no problem, just removed the else section. Thanks!

**Review by adongare — APPROVED** [review]

**Pull request merged**